litmus-portal/frontend/src/config/index.ts

```diff
@@ -1,7 +1,7 @@
 export default {
   environment: process.env.NODE_ENV,
-  api: {
-    url: process.env.REACT_APP_API_HOST || 'http://localhost:8080/api/v1',
+  analytics: {
+    url: 'https://staging-hub.litmuschaos.io/api/community',
```

Comment:
put this in a .env file under two section prod and staging

Reply:
I can add this to the k8s-manifest as env?

Reply:
Not to the k8s manifest. In frontend dir, put .env file for all the environment variable which will be used by the frontend.
you can call it by `process.env.ENV_VAR`

Reply:
I think we should keep the .env files for API secrets and config file is good for keeping pubic APIs...this setup was used by alden...and I liked the approach.

Reply:
Okay, for now we can take this directly. But in future we have to create .env and after chaos hub code merged to the master we will get the prod url.